### ISSUE TITLE
[Feature] Allow Customize Favicon override for each feed

### ISSUE THREAD

**Issue by math-GH** [FreshRSS/FreshRSS#6503]
### Discussed in https://github.com/FreshRSS/FreshRSS/discussions/6502

<div type='discussions-op-text'>

<sup>Originally posted by **johnsturgeon** May 27, 2024</sup>
I get several RSS feeds from reddit and the favicon is always the same (resulting in the SAME favicon for every feed).  It would be great if I could override the favicon on a feed by feed basis.

I did see a similar question here #3875 (...)</div>

**Comment by johnsturgeon** [comment]
Any update on this?

**Comment by math-GH** [comment]
> Any update on this?

Nothing new here

**Comment by bcotton** [comment]
As far as I can tell, the Reddit icon is not available in the RSS feed but rather in a subreddit-specific `JSON` URL.

https://www.reddit.com/r/redditdev/comments/dot8tn/how_can_i_get_the_icon_of_a_subreddit/

**Comment by tomasagray** [comment]
I agree, this would be a **really** nice feature to have. Icons help to organize articles in your mind when scanning. Just my 2c.

**Comment by Inverle** [comment]
implemented in #7646

**Comment by Alkarex** [comment]
@bcotton , @bcotton Tests welcome of https://github.com/FreshRSS/FreshRSS/pull/7646
